src/helpers/text_helper.ts

```diff
@@ -224,6 +224,40 @@ export function getContextFontSize(font: string): Pixel {
   return Number(font.match(pxRegex)?.[1]);
 }
 
+export function drawSplittedText(
+  context: CanvasRenderingContext2D,
+  text: string,
+  position: PixelPosition
+) {
+  const splitted = splitText(text);
+  const measure = context.measureText(splitted[1]);
+  context.fillText(splitted[1], position.x, position.y);
+  position.y -= measure.actualBoundingBoxAscent + measure.actualBoundingBoxDescent + 2;
+  context.fillText(splitted[0], position.x, position.y);
+}
+
+export function splitText(text: string): string[] {
+  let spaces = "";
+  while (text[0] === " ") {
+    spaces += " ";
+    text = text.slice(1);
+  }
```

Comment:
The idea here was to keep the leading space, in case of the user has put some starting space in the baseline description to have a spacing between the value and the spacing. As this is something that can be done for the moment, I didn't want to change this behavior, even for multiline description :)